src/ert/parsing/__init__.py

```diff
@@ -1,4 +1,27 @@
 from .config_errors import ConfigValidationError, ConfigWarning
+from .config_keywords import ConfigKeys
+from .config_schema import (
+    ConfigSchemaDict,
+    SchemaItemType,
+    init_site_config_schema,
+    init_user_config_schema,
+)
 from .lark_parser import parse as lark_parse
+from .types import Instruction
+from .workflow_job_keywords import WorkflowJobKeys
+from .workflow_job_schema import WorkflowJobSchemaDict, init_workflow_schema
 
-__all__ = ["lark_parse", "ConfigWarning", "ConfigValidationError"]
+__all__ = [
+    "lark_parse",
+    "ConfigWarning",
+    "ConfigValidationError",
+    "WorkflowJobKeys",
+    "WorkflowJobSchemaDict",
```

Comment:
`WorkflowJobSchemaDict` and `ConfigSchemaDict` is not used outside of ert.parsing so we don't need to expose this through `__init__.py`.